libraries/AP_Parachute/AP_Parachute.cpp

```diff
@@ -205,7 +211,7 @@ void AP_Parachute::check_sink_rate()
 // check settings are valid
 bool AP_Parachute::arming_checks(size_t buflen, char *buffer) const
 {
-    if (_enabled > 0) {
+    if (enabled()) {
```

Comment:
This doesn't look right.  If you have a switch but it is in the disabled position we should still be doing these checks.

We'll need a regression test for that so it doesn't happen in the future...

Reply:
Ahh agreed! I'll fix it. Thanks.